EntranceShuffle.py

```diff
@@ -409,7 +409,7 @@ def _add_boss_entrances():
 priority_entrance_table = {
     'Bolero': (['DMC Central Local'], ['OwlDrop', 'WarpSong']),
     'Nocturne': (['Graveyard Warp Pad Region'], ['OwlDrop', 'Spawn', 'WarpSong']),
-    'Requiem': (['Desert Colossus', 'Desert Colossus From Spirit Lobby'], ['OwlDrop', 'Spawn', 'WarpSong']),
+    'Requiem': (['Desert Colossus', 'Desert Colossus From Spirit Lobby'], ['OwlDrop', 'Spawn' 'WarpSong']),
```

Comment:
This needs to be reverted. In Python, `'Spawn' 'WarpSong'` is equivalent to `'SpawnWarpSong'`.

```suggestion
    'Requiem': (['Desert Colossus', 'Desert Colossus From Spirit Lobby'], ['OwlDrop', 'Spawn', 'WarpSong']),
```

Reply:
Yeah must have been a typo I missed